forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifierTest.java

```diff
@@ -27,6 +27,7 @@
 import com.forgerock.openbanking.jwt.services.CryptoApiClient;
 import com.forgerock.openbanking.model.Tpp;
 import com.forgerock.openbanking.model.oidc.OIDCRegistrationResponse;
+import org.junit.Ignore;
```

Comment:
Unused import

Reply:
Is there a way to get IntelliJ to automatically organise imports on save like Eclipse has?

Reply:
Found it! :-D

Reply:
Fixed